### DISCUSSION TITLE
Gene Symbol Placed After Colon on Some MSTRG IDs

### DISCUSSION THREAD

**Discussion by erichpeterson** [kvittingseerup/IsoformSwitchAnalyzeR#99] [Q&A] [answered]
Hi,
In the past, I had seen output from IsoformSwitchAnalyzeR that had a colon in between some of the MSTRG IDs and a gene symbol, i.e., MSTRG.1:TP53

I'm now not seeing that. I'm wondering in what circumstances are the colon and gene symbol added together?

Thanks

**Comment by kvittingseerup** [comment, marked as answer]
Thes with "joint" gene_id gene_names are the result of "rescuing" genes which were merged due to transcript assembly (via tools such as StringTie, Cufflinks etc). You can control the behaviour via the argument of the import function you have used. For the StringTie if a propper gene_id is pressent (e,g. 'ENSG') that is now used instead.

Cheers
Kristoffer
